files/steps/generate_dataset_from_hbase.py

```diff
@@ -120,22 +121,32 @@ def encrypt_plaintext(data_key, plaintext_string, iv=None):
     return ciphertext.decode("ascii"), iv.decode("ascii")
 
 
+def get_key_from_cache(kek):
```

Comment:
not sure this is necessary as a function - you can use the get() method of the dictionary, or just do this inline in get_plaintext_key()

Reply:
Suggestion has been implemented.